app/controllers/ad_units_controller.rb

```diff
@@ -49,7 +49,10 @@ def ad_unit_params
         :user_id,
         :rewarded,
         :interstitial,
-        :video_length
+        :video_length,
+        :click_url_android,
+        :click_url_ios,
+        :click_to_action
```

Comment:
Change `click_to_action` to `call_to_action` everywhere. You'll have to rollback your development (`rails db:rollback`) database as well as your test database (`rails db:rollback RAILS_ENV=test`) before changing the migration file and running `rails db:migrate` again.